server.py

```diff
@@ -38,23 +49,53 @@ def start(self):
     @cherrypy.tools.json_in()
     @cherrypy.tools.json_out()
     def move(self):
-        # This function is called on every turn of a game. It's how your snake decides where to move.
-        # Valid moves are "up", "down", "left", or "right".
-        # TODO: Use the information in cherrypy.request.json to decide your next move.
+        """
+        This function is called on every turn of a game. It's how your snake decides where to move.
+        Valid moves are "up", "down", "left", or "right".
+        
+        Use the information in 'data' to decide your next move. The 'data' variable can be interacted
+        with as a Python Dictionary, and contains all of the information about the Battlesnake board 
+        for each move of the game.
+
+        For a full example of 'data', see https://docs.battlesnake.com/references/api/sample-move-request
+        """
         data = cherrypy.request.json
+        my_head = data["you"]["head"]  # A dictionary of x/y coordinates like {"x": 0, "y": 0}
+        my_body = data["you"]["body"]  # A list of x/y coordinate dictionaries like [ {"x": 0, "y": 0}, {"x": 1, "y": 0}, {"x": 2, "y": 0} ]
+
+        # TODO: uncomment the lines below so you can see what this data looks like in your output!
+        # print(f"~~~ Turn: {data['turn']}  Game Mode: {data['game']['ruleset']['name']} ~~~")
+        # print(f"All board data this turn: {data}")        
+        # print(f"My Battlesnakes head this turn is: {my_head}")
+        # print(f"My Battlesnakes body this turn is: {my_body}")
 
-        # Choose a random direction to move in
         possible_moves = ["up", "down", "left", "right"]
-        move = random.choice(possible_moves)
 
-        print(f"MOVE: {move}")
+        # Don't allow your Battlesnake to move back in on it's own neck
+        possible_moves = server_logic.avoid_my_neck(my_head, my_body, possible_moves)
+
+        # TODO: Using information from 'data', find the edges of the board and don't let your Battlesnake move beyond them
+        # board_height = ?
+        # board_width = ?
+
+        # TODO Using information from 'data', don't let your Battlesnake pick a move that would hit its own body
+
+        # TODO: Using information from 'data', don't let your Battlesnake pick a move that would collide with another Battlesnake
+
+        # TODO: Using information from 'data', make your Battlesnake move towards a piece of food on the board
+
+        # Choose a random direction from the remaining possible_moves to move in, and then return that move
+        move = random.choice(possible_moves)
+        print(f"CHOOSING MOVE: {move} from all valid options in {possible_moves}")
```

Comment:
This is a clever future-proofing debug comment, where inevitably folks will accidentally remove all possible moves (via avoid wall/body etc), and maybe not notice, but this print statement will help them.

I just feel clever about this and wanted to point it out.